api/src/middleware/__tests__/competenciesRouter.ts

```diff
@@ -104,4 +109,80 @@ describe('competenciesRouter', () => {
         .expect(500, done);
     });
   });
+
+  describe('PUT /:id', () => {
+    it('should update a competency of user who authored it', done => {
+      const principalId = 2;
+      const competencyId = 3;
+      const label = 'label';
+      const description = 'description';
+      const competency = {
```

Comment:
This `competency` variable is declared but never used.  is it necessary in this test?

Reply:
It is used on line 131. That said, when a variable is only used once, a lot of times it's more readable to just inline the value, unless it's unclear what the value represents. Luckily, we have a lint rule that fails if a variable is declared and actually never used.

Reply:
This is meant to be used for testing update competency function. I will remove it since there is no need to test void functions.